src/west/main.py

```diff
@@ -199,6 +199,9 @@ def main(argv=None):
         argv = sys.argv[1:]
     args, unknown = parse_args(argv)
 
+    # Read the configuration files
```

Comment:
Should we try to create one for those using the old bootstrap?

Reply:
You mean people who have already bootstrapped?

Defaults are used if the configuration file doesn't exist, so shouldn't need it.